libpod/options.go

```diff
@@ -2339,6 +2340,18 @@ func WithVolatile() CtrCreateOption {
 		}
 
 		ctr.config.Volatile = true
+	}
```

Comment:
There is a compilation bug here.missing a return nil

Reply:
Fixed